### PR TITLE
Add Stride's stLUNA

### PR DESCRIPTION
**Pull request by shellvish**
Adds stLUNA to `cosmos/stride.json`

### CODE REVIEW

File: cosmos/stride.json [hunk 1]
```diff
@@ -49,7 +49,11 @@
       "coinDenom": "stSTARS",
       "coinMinimalDenom": "stustars",
       "coinDecimals": 6
-    }
+    },
+      "coinDenom": "stLUNA",
+      "coinMinimalDenom": "stuluna",
+      "coinDecimals": 6
+    },
```
Comment (HeesungB) [inline cosmos/stride.json:56]: Could you remove comma(,) line 56?
It doesn't fit the json format.

Comment (shellvish) [reply cosmos/stride.json]: Removed! How does this look?

File: cosmos/stride.json [hunk 2]
```diff
@@ -49,6 +49,10 @@
       "coinDenom": "stSTARS",
       "coinMinimalDenom": "stustars",
       "coinDecimals": 6
+    },
+      "coinDenom": "stLUNA",
```
Comment (HeesungB) [inline cosmos/stride.json:53]: @shellvish Could you add `{` between 52 and 53 lines?

Comment (shellvish) [reply cosmos/stride.json:54]: Wow, so sorry, not sure how I missed that. Thanks a lot for the flag!

**Comment by HeesungB** [comment]
@shellvish Thank you for contribution!

**Pull request merged**